Tests/Apollo/Network/HttpNetworkTransportTests.swift

```diff
@@ -0,0 +1,342 @@
+import XCTest
+@testable import Apollo
+import ApolloTestSupport
+import StarWarsAPI
+
+class HttpNetworkTransportTests: XCTestCase {
+
+  private final let endpoint = "http://localhost:8080/graphql"
+  
+  func testRequestBody() {
+    let network = HTTPNetworkTransport(url: URL(string: endpoint)!)
+    let mockSession = MockURLSession()
+    network.session = mockSession
+    let query = HeroNameQuery()
+    let _ = network.send(operation: query) { _,_ in }
+    let request = mockSession.lastRequest
+    
+    XCTAssertNotNil(request)
+    XCTAssert(request!.url?.host == network.url.host)
```

Comment:
Heads up that failed force-unwraps in tests crash the whole test suite rather than failing the test. Generally I prefer to do this instead: 

```swift
guard let foo = bar else {
   XCTFail("No value for `bar`!")
   return 
}
```

That way the test with a nil value fails with a clear error message and other tests can still run.

Reply:
updated and other too.